src/style.css

```
@@ -0,0 +1,140 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: normal;
+  color: #060b35;
+  font-size: 10px;
+}
+
```

I think you should add 
```suggestion
* {
box-sizing: border-box;
}
```